test/integration/transactions/transactions.spec.test.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+
+import { loadSpecTests } from '../../spec';
+import { runUnifiedSuite } from '../../tools/unified-spec-runner/runner';
+
+const SKIPPED_TESTS = [
+  // TODO(NODE-5925) - secondary read preference not allowed in transactions.
+  'readPreference inherited from defaultTransactionOptions',
+  // TODO(NODE-5924) - Fix modification of readConcern object post message send.
+  'readConcern local in defaultTransactionOptions',
+  'defaultTransactionOptions override client options',
+  'transaction options inherited from defaultTransactionOptions',
+  'transaction options inherited from client',
+  'causal consistency disabled'
+  // TODO(NODE-5855) - Gone away after NODE-5929
```

Comment:
The unified tests that are related to NODE-5855 are automatically skipped by the unified tests due to the fact that `targetedFailPoint` will not work with multiple mongoses in load balancer mode. So those tests only run on sharded clusters.